libs/sdk-ui-dashboard/src/model/store/index.ts

```diff
@@ -181,9 +181,12 @@ export { selectLegacyDashboards } from "./legacyDashboards/legacyDashboardsSelec
 
 export type { UndoEnhancedState, UndoEntry } from "./_infra/undoEnhancer.js";
 export type { LayoutState, LayoutStash } from "./layout/layoutState.js";
+
+export { layoutActions } from "./layout/index.js";
```

Comment:
Should this be exposed? I don't see other store actions being exposed directly.

Reply:
replaced by command+event combo